src/containers/toast/toast.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { ToastContainer } from 'react-toastify';
+
+import { useSelector } from 'react-redux';
+
+const Toast = () => {
+  const toastSettings = useSelector((state) => state.Toast.toastSettings);
```

Comment:
Please move this selector to separate file.

Reply:
done

Reply:
done